RootChannels.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+
+namespace RestoreFromLocal
+{
+    public class Topic
+    {
+        public string value { get; set; }
+        public string creator { get; set; }
+        public int last_set { get; set; }
```

Comment:
For this type of property, use `JsonProperty` attribute and make the property name upper camel case.